opencti-platform/opencti-front/src/private/components/cases/CaseTasksLine.tsx

```diff
@@ -0,0 +1,183 @@
+  },
+  placeholder: {
+    display: 'inline-block',
+    height: '1em',
+    backgroundColor: theme.palette.grey?.[700],
+  },
+  paper: {
+    height: '100%',
+    minHeight: '100%',
+    margin: '-5px 0 0 0',
+    padding: 0,
+    borderRadius: 6,
+  },
+}));
+
+const dataColumns: DataColumns = {
+  name: {
+    label: 'Title',
+    width: '25%',
+    isSortable: true,
+    render: (task: CaseTasksLine_data$data) => task.name,
+  },
+  dueDate: {
+    label: 'Due Date',
+    width: '15%',
+    isSortable: true,
+    render: (task: CaseTasksLine_data$data, { fldt }) => fldt(task.dueDate),
+  },
+  objectAssignee: {
+    label: 'ObjectAssignee',
+    width: '25%',
+    isSortable: true,
+    render: (task: CaseTasksLine_data$data) => task.objectAssignee?.edges?.map(({ node }) => node.name).join(', '),
+  },
+  objectLabel: {
+    label: 'Labels',
+    width: '20%',
+    isSortable: true,
+    render: (task: CaseTasksLine_data$data) => (
+      <StixCoreObjectLabels
+        variant="inList"
+        labels={task.objectLabel}
+      />
+    ),
+  },
+  x_opencti_workflow_id: {
+    label: 'Status',
+    width: '15%',
+    isSortable: true,
+    render: (task) => (
+      <ItemStatus
+        status={task.status}
+        variant="inList"
+        disabled={!task.workflowEnabled}
+      />
+    ),
+  },
+};
+
+const CaseTaskFragment = graphql`
+  fragment CaseTasksLine_data on CaseTask{
+    id
+    name
+    dueDate
+    description
+    workflowEnabled
+    objectMarking {
+      edges {
+        node {
+          definition
+          definition_type
+          id
+        }
+      }
+    }
+    objectLabel {
+      edges {
+        node {
+          id
+          value
+          color
+        }
+      }
+    }
+    objectAssignee {
+      edges {
+        node {
+          entity_type
+          id
+          name
+        }
+      }
+    }
+    status {
+      template {
+        name
+        color
+      }
+    }
+  }
+`;
+
+interface CaseTasksLineProps {
+  data: CaseTasksLine_data$key
+  paginationOptions: CaseTasksLinesQuery$variables
+  caseId: string
+}
+
+const CaseTasksLine: FunctionComponent<CaseTasksLineProps> = ({ data, paginationOptions, caseId }) => {
+  const classes = useStyles();
+  const { fldt } = useFormatter();
+
+  const task = useFragment(CaseTaskFragment, data);
+
+  return (
+    <ListItem
+      classes={{ root: classes.item }}
+      divider={true}
+    >
+      <ListItemIcon classes={{ root: classes.itemIcon }}>
+        <ItemIcon type={'Case-Task'}></ItemIcon>
+      </ListItemIcon>
+      <ListItemText primary={
+        <div>
+          {Object.values(dataColumns).map((value) => (
+            <div
+              key={value.label}
+              className={classes.bodyItem}
+              style={{ width: value.width }}
+            >
+              {value.render?.(task, { fldt })}
+            </div>
+          ))}
+        </div>
+      }
+      />
+      <ListItemIcon classes={{ root: classes.goIcon }}>
```

Comment:
where is this class goIcon ? There also seem to be unused classes in this file : itemIconDisabled, paper, placeholder